pkg/controllers/disruption/types.go

```diff
@@ -175,3 +175,18 @@ func (c Command) LogValues() []any {
 		"replacement-nodes", replacementNodes,
 	}
 }
+
+// SetDisruptionCost sets the disruption cost of a candidate for testing
```

Comment:
Do you need these getters and setters now?

Reply:
Moved these to public on `Candidate`